addons/web/tests/test_assets.py

```diff
@@ -65,3 +69,18 @@ def test_assets_generate_time(self):
         for bundle, duration in self.generate_bundles():
             threshold = thresholds.get(bundle, 2)
             self.assertLess(duration, threshold, "Bundle %r took more than %s sec" % (bundle, threshold))
+
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestLoad(HttpCase):
+    def test_load_website(self):
+        self.authenticate('admin', 'admin')
+        _save_attachment = odoo.addons.base.models.assetsbundle.AssetsBundle.save_attachment
+        def save_attachment(bundle, extension, content):
+            attachment = _save_attachment(bundle, extension, content)
+            _logger.error("Trying to save an attachement for %s when it should already exist: %s", bundle.name, attachment.url)
+            return attachment
+
+        with patch('odoo.addons.base.models.assetsbundle.AssetsBundle.save_attachment', save_attachment):
+            self.url_open('/web')
```

Comment:
```suggestion
        with patch('odoo.addons.base.models.assetsbundle.AssetsBundle.save_attachment') as save_attachment:
            res = self.url_open('/web')
            save_attachment.assert_not_called()
            res.raise_for_status()
            
```

Reply:
If you really want to save the attachment nonetheless, simply add `save_attachment.side_effect = _save_attachment` before `url_open()`

Reply:
The purpose of the test is to check that we are not trying to generate the assets during a simple http test, saving it is not useful since it will be rollback anyway.  But this may be cleaner

Reply:
Finaly went back to the initial solution since the message is easier to understand.